cypress/integration/ut2019.spec.js

```diff
@@ -1,14 +1,55 @@
-describe('ut2019', function() {
-	const email = 'test.user@ut2019.ee';
-	const name = 'Test User';
-	const message = 'Custom message!';
-	it('Submit form successfully', function() {
-		cy.clearCookies();
-		cy.visit('https://ut2019.voog.com/contact');
-		cy.get('.contact-name input').type(name);
-		cy.get('.contact-email input').type(email);
-		cy.get('.contact-message textarea').type(message);
-		cy.get('input[type="submit"]').click();
-		cy.get('.form_notice').should('have.text', 'Thank you!');
-	});
-});
+//List of different language options
+describe('generated from a list', () => {
+    const operations = [{
+            value: "et",
+        },
+        {
+            value: "en",
+        },
+    ]
+
+    operations.forEach((operation) => {
+
+        describe("ut2019", function() {
+            const email = "test.user@ut2019.ee";
+            const name = "Test User";
+            const message = "Custom message!";
+            it("Submit form successfully", function() {
+                cy.clearCookies();
+                cy.visit('https://ut2019.voog.com/' + operation.value);
+                cy.get('.menu-main__list > :nth-child(2) > a').click();
+                cy.get(".contact-name input").type(name);
+                cy.get(".contact-email input").type(email);
+                cy.get(".contact-message textarea").type(message);
+                cy.get('input[type="submit"]').click();
+                cy.get(".form_notice")
+            });
+        });
+
+        describe("ut2019", function() {
+            const invalidEmail = "test.@";
+            const invalidName = "1234";
+            const invalidMessage = " ";
+            it("Submit ivalid form", function() {
+                cy.clearCookies();
+                cy.visit('https://ut2019.voog.com/' + operation.value);
+                cy.get('.menu-main__list > :nth-child(2) > a').click();
```

Comment:
Using this kind of selector  for clicking on the Contact link is a bad practice. What happens when a new menu item is added to the page before the Contact link? Instead you should here find the element by text "Contact" or by href attribute value "/contact" or if you are opening the contacts page from a url then just use cy.visit to go directly to the contacts page.